code/modules/jobs/job_types/clown.dm

```diff
@@ -18,6 +18,16 @@
 	display_order = JOB_DISPLAY_ORDER_CLOWN
 	departments = DEPARTMENT_SERVICE
 
+	mail_goodies = list(
+		/obj/item/food/grown/banana = 100,
+		/obj/item/food/pie/cream = 50,
+		/obj/item/clothing/shoes/clown_shoes/combat = 10,
+		/obj/item/reagent_containers/spray/waterflower/lube = 20, // lube
+		/obj/item/melee/transforming/energy/sword/bananium = 1,
+		/obj/item/shield/energy/bananium = 1,
```

Comment:
oh god, there's a reason why this shift is clown ops only

if someone other than the clown gets it, RIP station

Reply:
This is a holdover from the last PR, but in hindsight yeah you're totally right.